app/filters/filters.js

```diff
@@ -345,4 +345,15 @@ angular.module('portainer.filters')
   return function (createdBy) {
 	  return createdBy.replace('/bin/sh -c #(nop) ', '').replace('/bin/sh -c ', 'RUN ');
   };
-});
+})
+.filter('trimshasum', function () {
+  'use strict';
+  return function (imageName) {
+    if (imageName) {
+      var name = imageName.split('sha')[0];
```

Comment:
I think that we should split on `sha256` instead of `sha` because some organisations might be prefixed by `sha`.

Reply:
Why not `sha256:`?

Reply:
Yup, even better indeed.